readme.md

```diff
@@ -28,35 +28,19 @@ Please make sure you install the following tools before starting with the instal
 
 ## Installation
 
-> Note that you're free to adjust the `~/Sites/pastebin` location to any directory you want on your machine.
+> Note that you're free to adjust the `~/Code/pastebin` location that will be in the `Homestead.yaml` to any directory you want on your machine.
```

Comment:
This section needs to be re-worded. 

The Homestead `make` command will automatically figure out where it is on the filesystem and map the current project to the full filesystem path. This means the folder mapping that you see in the `Homestead.yaml.example`:

```
folders:
    - { map: ~/Sites/pastebin, to: /home/vagrant/pastebin }
```

Will get transformed to:

```
folders:
    -
        map: /Users/halo/PhpstormProjects/pastebin
        to: /home/vagrant/Code/pastebin
```

(For me because `/Users/halo/PhpstormProjects/` is where I keep my projects, the path you will see is unique to that computer)